src/Renderer/PointsMaterial.js

```diff
@@ -42,9 +60,23 @@ class PointsMaterial extends RawShaderMaterial {
         if (__DEBUG__) {
             this.defines.DEBUG = 1;
         }
+
         this.updateUniforms();
     }
 
+    copy(source) {
+        super.copy(source);
+        if (source.uniforms.projectiveTextureAlphaBorder) {
```

Comment:
Why would you need to do that ? This should be handled by the implementation of `ShaderMaterial.copy`.

Reply:
Because, it's a link material to oriented image material, it's necessary to reference these uniforms.